civitai-downloader-v2/src/cli/main.py

```
@@ -114,623 +114,349 @@ def cli(ctx, config, verbose):
+            click.echo(f"Error during search: {e}", err=True)
+            return
+        finally:
+            # Close streaming exporters
+            try:
+                # Initialize variables if they don't exist
+                csv_exporter = locals().get('csv_exporter')
+                json_exporter = locals().get('json_exporter')
+                
+                if csv_exporter:
+                    csv_exporter.close()
+                    click.echo(f"Streaming CSV export completed: {search_output}")
```

Ruff (0.12.2)

Undefined name `search_output`

(F821)